### ISSUE TITLE
Existing secrets' key names should be made more explicit in the README

### ISSUE THREAD

**Issue by bjd2385** [DataDog/helm-charts#366]
**Describe what happened:**

I regularly use `SealedSecret`s to manage secrets in a more secure fashion, as opposed to standard Secrets. To use these, I point the `ap[ip]KeyExistingSecret` fields at the `SealedSecret`'s name.

First time around, I assumed the `key` names to be `ap[ip]Key` and it resulted in a failed deployment.

**Describe what you expected:**

Ultimately that it be made more clear that the Secret key names need to be `api-key` and `app-key`. Currently in the `README` (at least up to the current release 2.21.3), `api-key` is mentioned only once, buried in another command, and `app-key` isn't mentioned at all.

**Steps to reproduce the issue:**

```
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/cluster-wide: "true"
  creationTimestamp: null
  name: datadog-secret
  namespace: {{ .Values.namespace | default .Release.Namespace }}
spec:
  encryptedData:
    apiKey: {{ .Values.secrets.sealedSecretApi }}
    appKey: {{ .Values.secrets.sealedSecretApp }}
  template:
    metadata:
      annotations:
        sealedsecrets.bitnami.com/cluster-wide: "true"
      name: datadog-secret
      namespace: {{ .Values.namespace | default .Release.Namespace }}
    type: Opaque
```
this obviously will fail a deployment if you point both `ap[ip]KeyExistingSecret` fields at `datadog-secret`, since the expected key names in the templates are `api-key` and `app-key`.

**Additional environment details (Operating System, Cloud provider, etc):**

I'm running K8s v1.21, Datadog chart v2.19.9, but this doesn't appear to have changed between this release and the current (2.21.3) release.

Somewhat related to #60, but less of a how-to, and more of a "hey it'd be nice to have". Happy to submit a PR to try to make this more obvious when I have more time. (E.g., might want to state these key names in the `datadog.apiKeyExistingSecret` and `datadog.appKeyExistingSecret` boxes, and/or elsewhere in an example, to ensure others don't trip over this as well.)

**Comment by clamoriniere** [comment]
Hello @bjd2385
Thanks for opening this issue. Indeed we should improve our documentation about the attended keys for the "API key" and the "APP Key". 
Feel free to open a pull request to improve our documentation.
